### PR TITLE
Made more minor changes from cb_types

### PR DESCRIPTION
**Pull request by Ramit110**

### CODE REVIEW

File: common/cb_types.cwt [hunk 1]
```diff
@@ -49,6 +49,7 @@ casus_belli = {
     ### UNKNOWN. Defaults to "no".
     is_independence = bool
 
+    ## cardinality = 0..1
```
Comment (UristMcDorf) [inline common/cb_types.cwt:52]: Update the docs for the default value then. Ditto for the rest of similar stuff.

Comment (Ramit110) [reply common/cb_types.cwt:52]: Aye. 
Checking the defaults.

File: triggers.cwt [hunk 2]
```diff
@@ -1965,6 +1969,10 @@ alias[trigger:prestige] = int
 ### Checks if the scoped character is imprisoned.
 alias[trigger:prisoner] = bool
 
+## scope = character
+### Checks if the scoped character is a captor of another character.
```
Comment (UristMcDorf) [inline triggers.cwt]: `True if the scoped character is the captor of the given character.` probably.

File: triggers.cwt [hunk 3]
```diff
@@ -2454,6 +2470,10 @@ alias[trigger:religion_group] = scope[religion]
 ### True if scoped religion belongs to the same religion group as the given entity.
 alias[trigger:religion_group] = scope[religion]
 
+## scope = { character province }
+### True if scoped entity has a religion that belongs to the same religion group as the given entity.
```
Comment (UristMcDorf) [inline triggers.cwt:2474]: the scoped entity

Comment (Ramit110) [reply triggers.cwt]: Eh?

**Comment by tboby** [comment]
All looks good to me :)

**Comment by Ramit110** [comment]
Then we pushing this bois?

**Comment by Ramit110** [comment]
(Lol just want to work on other bits and pieces)

**Pull request merged**